process/attendance.md

```diff
@@ -0,0 +1,27 @@
+# Attending a WebAssembly meeting
+
+Meetings are held in-person at the location determined in the meeting
+invitation. The invitation will contain logistics on location, time, address,
+parking, nearby hotels, Wi-Fi, and host contact. This invitation will be sent no
+later than 2 weeks before the meeting, through a pull request on the
+[meetings repository](https://github.com/WebAssembly/meetings) as well as
+through an email to the Community Group's mailing list.
+
+It is expected that a small breakfast, snacks, and beverages will be offered on
+premise, either for free or for a reasonable fee. The host may organize a dinner
+as well. If a dinner is held, the host may require RSVP ahead of time.
+
+All attendees are required to be members of the
+[community group](w3.org/community/webassembly/) prior to attending. Further, we
+request that attendees register in advance so that the host may plan capacity
+accordingly and so that the meeting notes may record who was present.
+
+The host must not request that an NDA be signed by attendees.
```

Comment:
This might limit the range of possible hosts.
Though I suppose not having it would limit attendees :-)